### PR TITLE
Proposal store fix

### PR DESCRIPTION
**Pull request by jreidinger**
[bsc#994127](https://bugzilla.suse.com/show_bug.cgi?id=994127)

### CODE REVIEW

**Comment by mvidner** [comment]
Importantly, Suresh said in the bug that an analogous fix (changing his clients to return `nil`) has fixed the problem.

**Comment by jreidinger** [comment]
@mvidner yep, this fix just ensure that doc and code is in sync, so both ways can be used.

File: package/yast2-installation.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Aug 18 11:52:39 UTC 2016 - jreidinger@suse.com
+
+- fix skipping of proposal returning empty hash otherwise it
+  creates non sense proposal entry in UI (bnc#994127)
```
Comment (mvidner) [inline package/yast2-installation.changes]: I am not sure people will understand that without context. What about this:

> proposal entries were always displayed (with a  foo_proposal heading) even if empty hash was returned (bnc#994127)

**Comment by mvidner** [comment]
I have changed the target to SP2 and made it mergeable.

**Comment by mvidner** [comment]
LGTM

**Comment by jreidinger** [comment]
@mvidner thanks

**Pull request merged**